server/controller/routes/house/updateHouse.js

```diff
@@ -0,0 +1,39 @@
+const { updateHouse } = require('../../../database/queries/house');
+
+const updateHouseController = async (req, res, next) => {
+  try {
+    const { houseId } = req.params;
+    const {
+      roomNo,
+      category,
+      price,
+      description,
+      title,
+      locationId,
+      bathNo,
+      area,
+      image,
+    } = req.body;
```

Comment:
you can pass the object req. body directly to the query, instead of distracting then  pass it